optaweb-vehicle-routing-backend/src/main/java/org/optaweb/vehiclerouting/plugin/planner/SolverManager.java

```diff
@@ -52,28 +56,29 @@
  * <li>Listens for best solution changes and publishes new best solutions via {@link RouteChangedEventPublisher}.</li>
  * </ul>
  */
-@Component("optaweb-solver-manager")
+@ApplicationScoped
+@Default
 class SolverManager implements SolverEventListener<VehicleRoutingSolution> {
```

Comment:
Probably should be renamed (to not be confused with `optaplanner.core.api.solver.SolverManager`

Reply:
#453